auth-api/src/auth_api/services/affiliation.py

```diff
@@ -429,6 +429,7 @@ def fix_stale_affiliations(org_id: int, entity_details: Dict, **kwargs):
         identifier: str = entity_details.get("identifier")
         current_app.logger.debug(f"<fix_stale_affiliations - {nr_number} {bootstrap_identifier} {identifier}")
         from_entity: Entity = EntityService.find_by_business_identifier(nr_number, skip_auth=True)
+        temp_entity: Entity = EntityService.find_by_business_identifier(bootstrap_identifier, skip_auth=True)
```

Comment:
remove this type hint it should be inferred from EntityService.find_by_business_identifier on the method def